lib/DefinePlugin.js

```diff
@@ -83,6 +83,9 @@ const toCode = (code, parser) => {
 	if (typeof code === "object") {
 		return stringifyObj(code, parser);
 	}
+	if (typeof code === "bigint") {
+		return `${code}n`;
```

Comment:
This should depend on the `ecmaVersion` option and either generate `123n` or `BigInt("123")`